src/scripts/components/Debug.js

```diff
@@ -3,20 +3,19 @@ import styled from 'styled-components'
 import { paratii } from '../utils/ParatiiLib'
 import type { RootState } from 'types/ApplicationTypes'
 
-const DebugBox = styled.header`
-  background-color: #fff;
-  border: 2px dashed red;
+const DebugBox = styled.div`
+  background-color: #ccc;
+  border: 4px dashed #f55;
```

Comment:
I think we should avoid any hex colors directly in components and instead, if we have new colors, define them in our colors module

Reply:
i agree but this is just a temporary ugly debug box, just for dev